### DISCUSSION TITLE
What features are NOT available without any ring protect subscription plan ?

### DISCUSSION THREAD

**Discussion by rezzalex** [tsightler/ring-mqtt#874] [Support Q&A] [answered]
I just bought new ring cams and just understand that I own the protect plan for free for 30 days...
And I am not sure of what I will get after ?
No recording at all ?
No snapshot at all ?
Stream live view ?
Event live view ?

**Comment by tsightler** [comment, marked as answer]
These items are generally spelled out in the Wiki and FAQ.  Summary version is:

> No recording at all ?

No recording in the Ring cloud at all.  You can use automations to manually record streams locally, but recording won't start until after the motion is received, so this has some significant downsides vs a subscription.

> No snapshot at all ?

Snapshot limitations are well covered in the FAQ and vary based on the exact model of device.  Summary is as follows:
1) All devices can take snapshots when idle, so interval snapshots generally work.
2) **Low-power devices** (Most non-Pro doorbells, anything running on battery or "trickle" charge devices, solar, etc) -- Snapshot are possible only when the device is otherwise idle.  With no subscription, Ring notifications do not include the snapshot image that was taken when the motion was triggered, so you will not receive an updated snapshot on motion.  Some users work around this by recording a short video and sending that as a notification, but this will be delayed 5-10 seconds at least.
3) **High-power devices** (Pro Doorbells and most devices that use mains power or PoE) -- These devices can take a snapshot at any time, even while recording during motion, so snapshots can still be taken after motion events are detected, but these snapshots will be delayed vs the native snapshots available in Ring rich notifications since the snapshot will not be requested until after ring-mqtt receives the motion/ding notificaiton.

> Stream live view ?


Always possible

> Event live view ?

Not possible, since there are no event recordings in the Ring cloud to play back.

To be completely clear, ring-mqtt is not a project intended to replace the requirement for a Ring subscription, rather, the capabilities of ring-mqtt are significantly enhanced if you do have a subscription, allowing for faster notification, and the same snapshot image as provided natively in the Ring app, playback of prior events, etc.  However, if you can live with the limitations stated above for your device, then it's certainly possible to use ring-mqtt without a subscription for cameras.

**Reply by rezzalex** [reply]
Thanks a lot for your quick answer and your project.

I was not clear on all that before purchasing.

I will return the cameras and use another brand.
Thanks a lot anyway.
